docs/setup-cluster/security/tls.rst

```diff
@@ -75,3 +75,247 @@ untrusted certificate and display a hash of the certificate. You may wish to con
 your system administrator; in any case, if you confirm the connection to the master, the certificate
 will be stored on the computer where the CLI is being run and future connections to the master will
 be made without confirmation.
+
+*************************************
+ Let's Encrypt TLS Certificate Setup
+*************************************
+
+This section describes how to set up the Let's Encrypt TLS certificate.
+
+Installing a Let's Encrypt TLS certificate requires an ACME protocol client ``certbot`` and one of
+the following domain token verification methods:
+
+-  HTTP-01, or
+-  DNS-01
+
+.. note::
+
+   For more information about the domain token verification methods, visit `Let's Encrypt
+   documentation <https://letsencrypt.org/docs/challenge-types/>`_.
+
+Prerequisites
+=============
+
+You must have ``snapd`` installed to use ``certbot``.
+
+Installing Snapd and Certbot
+============================
+
+This section provides information about installing ``snapd`` and ``certbot`` and adding EPEL to RHEL
+8 or CentOS 8.
+
+The following websites provide more information about installing ``snapd`` and ``certbot``:
+
+-  `Installing snap on Red Hat Enterprise Linux (RHEL)
+   <https://snapcraft.io/docs/installing-snap-on-red-hat>`_
+-  `Installing snap on CentOS <https://snapcraft.io/docs/installing-snap-on-centos>`_
+-  `certbot instructions <https://certbot.eff.org/instructions?ws=other&os=centosrhel8>`_
+
+Adding EPEL to RHEL 8
+---------------------
+
+To add the EPEL repository to a RHEL 8 system, run the following commands:
+
+.. code:: bash
+
+   sudo dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+   sudo dnf upgrade
+
+Adding EPEL to CentOS 8
+-----------------------
+
+To add the EPEL repository to a CentOS 8/9 Stream system, run the following commands:
+
+.. code:: bash
+
+   sudo dnf install epel-release
+   sudo dnf upgrade
+
+Installing Snapd Software
+-------------------------
+
+To install ``snapd``, run the following commands:
+
+.. code:: bash
+
+   sudo yum install snapd
+   sudo systemctl enable --now snapd.socket
+   sudo ln -s /var/lib/snapd/snap /snap
+
+.. note::
+
+   On Debian/Ubuntu, ``snapd`` is usually already installed.
```

Comment:
```suggestion
   On Debian/Ubuntu, snapd is usually already installed.
```
And if snapd turns out not to be necessary on Debian/Ubuntu, then remove this line entirely.